mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1259,7 +1267,11 @@ trait Completions { this: MetalsGlobal =>
           "match",
           new l.TextEdit(
             editRange,
-            "match {\n\tcase$0\n}"
+            if (clientSupportsSnippets) {
+              "match {\n\tcase$0\n}"
+            } else {
+              "match"
```

Comment:
avoid the braces if we have no snippets. Inserting braces will result in a cursor out of place (`{}<CURSOR>`), which seems bad.

Reply:
From my point of view, it is bad to end up with `foo{}<CURSOR>` after completion. I ran into a similar case with Apple's swift server and it forced me to do [this](https://github.com/bstaletic/ycmd/blob/swift/ycmd/completers/swift/swift_completer.py#L106-L110) - overriding completion algorithm for swift to strip `()` from `foo()`, even if there's something in between `(` and `)`.

Reply:
thanks for the feedback! I'm actively avoiding that pattern, except in some cases where the completion is still very useful and/or it's opt-in (e.g. it's not the default case)